release-notes/columnstore/columnstore-1-0/mariadb-columnstore-101-alpha-release-notes.md

```diff
@@ -17,7 +17,7 @@ To understand what to expect from this release, a little background is needed:[M
 We have not yet had time to ourselves extensively test all [MariaDB ColumnStore](https://github.com/mariadb-corporation/docs-release-notes/blob/test/kb/en/mariadb-columnstore/README.md) features and MariaDB ColumnStore has not yet been tested by a larger community, so there may be some issues that needs to be fixed in the next releases. We do however expect that most things should work and be reasonably stable.
 
 Please provide feedback in [JIRA](https://jira.mariadb.org/browse/MCOL) for anything that is not working as expected so that we can fix it before we make the release available for the larger community.\
-For general "how to questions" ask questions [here](https://mariadb.com/kb/en/mariadb/mariadb-columnstore/) or subscribe to mariadb-columnstore@googlegroups.com
+For general "how to questions" ask questions [here](https://app.gitbook.com/o/diTpXxF5WsbHqTReoBsS/s/rBEU9juWLfTDcdwF3Q14/) or subscribe to mariadb-columnstore@googlegroups.com
```

Comment:
gitbook: This is a create a new organisation link.

googlegroups could be a link too.